nullaway/src/main/java/com/uber/nullaway/ErrorMessage.java

```diff
@@ -1,11 +1,38 @@
 package com.uber.nullaway;
 
+import static com.uber.nullaway.NullAway.INITIALIZATION_CHECK_NAME;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.AnnotationTree;
+import com.sun.source.tree.ClassTree;
+import com.sun.source.tree.MethodInvocationTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.ModifiersTree;
+import com.sun.source.tree.Tree;
+import com.sun.source.tree.VariableTree;
+import com.sun.source.util.TreePath;
+import com.sun.tools.javac.code.Symbol;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.StreamSupport;
+import javax.annotation.Nullable;
+import javax.lang.model.element.Element;
+
 /** Contains error message string to be displayed and the message type from {@link MessageTypes}. */
 public class ErrorMessage {
+
+  static Config config;
+  static NullAway nullAway;
```

Comment:
No static fields please; they can lead to memory leaks and data races.  Also we shouldn't be keeping a pointer to the `NullAway` object here.  We should pass in just the state that we need.

Reply:
Done 👍 
Separated the code into new `ErrorBuilder` class. 
   